demisto_sdk/commands/generate_docs/generate_integration_doc.py

```diff
@@ -1,21 +1,42 @@
 import os.path
+import re
+from typing import Optional, Tuple
 
-from demisto_sdk.commands.common.tools import (get_yaml, print_error,
+from demisto_sdk.commands.common.tools import (LOG_COLORS, get_yaml,
+                                               print_color, print_error,
                                                print_warning)
 from demisto_sdk.commands.generate_docs.common import (
     add_lines, build_example_dict, generate_numbered_section, generate_section,
     generate_table_section, save_output, stringEscapeMD)
 
 
+def append_or_replace_command_in_docs(old_docs: str, new_str: str, command_name: str) -> Tuple[str, list]:
+    regexp = rf'(?:###\s{command_name}).+?(?:(?=(?:\n###\s))|(?=(?:\n##\s))|\Z)'
+    # Read doc content
+    errs = list()
+    if re.fullmatch(regexp, old_docs, flags=re.DOTALL):
+        new_docs = re.sub(regexp, new_str, old_docs, flags=re.DOTALL)
+        print_color('New command docs has been replaced in file.', LOG_COLORS.GREEN)
+    else:
+        if command_name in old_docs:
+            errs.append('Could not replace the command in the file although it is presented in the file.'
```

Comment:
did you encounter any doc that got here?

Reply:
it could be, it's a safe way.